tools/c7n_azure/tests/test_tags.py

```diff
@@ -43,27 +74,6 @@ def test_add_tags(self, update_resource_tags):
         expected_tags.update({'tag3': 'value3'})
         self.assertEqual(tools.get_tags_parameter(update_resource_tags), expected_tags)
 
-    @patch('c7n_azure.tags.TagHelper.update_resource_tags')
-    def test_add_tags_from_resource(self, update_resource_tags):
```

Comment:
Why do you remove it?

Reply:
Because the TagsHelper class doesn't know about the Lookup anymore :)

Reply:
I think those are good tests to have, so we can move them to the tag action

Reply:
Agreed! I added them back to the action test!